runtime/doc/quickfix.txt

```diff
@@ -1564,20 +1564,24 @@ recognized as a command separator.  The backslash before each space is
 required for the set command.
 
 					    *cfilter-plugin* *:Cfilter* *:Lfilter*
-If you have too many matching messages, you can use the cfilter plugin to
-reduce the number of entries.  Load the plugin with: >
+If you have too many entries in a quickfix list, you can use the cfilter
```

Comment:
Doc change is from v8.1.1281.

Reply:
This section was moved below `:ldo`. Check the patch.

Reply:
Updated